packages/platform/atoms/hooks/event-types/useGetEventTypeById.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { BASE_URL, API_VERSION, V2_ENDPOINTS, SUCCESS_STATUS } from "@calcom/platform-constants";
+import type { EventType as AtomEventType } from "@calcom/platform-libraries";
+import type { ApiResponse, ApiSuccessResponse } from "@calcom/platform-types";
+
+import http from "../../lib/http";
+
+export const QUERY_KEY = "get-event-by-id";
+export const useGetEventTypeById = (id: number | null) => {
```

Comment:
this does not seem to be used in this pr ?

Reply:
Yeah this was mainly for future reference in case we might need a hook like this.